test/spec/ui/viewer_spec.js

```diff
@@ -232,6 +232,9 @@ describe('ui.viewer.Viewer', function () {
                 text: "Rabbits with cloaks"
             };
             v.load([ann]);
+            window.confirm = function () {
```

Comment:
Almost there!!

Instead of using `window.confirm =` you just need to use `sandbox.stub(window, 'confirm').returns(true)`